src/RaygunAspireWebApp/Controllers/IngestionController.cs

```diff
@@ -43,5 +53,22 @@ public async Task<IActionResult> Entries()
 
       return Accepted();
     }
+
+    private void EnforceRetentionAsync()
+    {
+      var files = Directory.GetFiles(ErrorsFolderPath)
+            .Select(filePath => new FileInfo(filePath))
+            .OrderBy(filePath => filePath.CreationTime).ToList();
```

Comment:
You might be able to just .Skip(1000) and delete the  rest.

Reply:
Sure can, I've made this simplification.